assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/stable_diffusion/constants.py

```diff
@@ -0,0 +1,29 @@
+from enum import Enum
+
+class _CustomEnum(Enum):
+    @classmethod
+    def has_value(cls, value):
+        return value in cls._value2member_map_
+
+
+class Tasks(_CustomEnum):
+    """
+    Task types supported by stable diffusion
+    """
+    TEXT_TO_IMAGE = "text-to-image"
+    INPAINTING = "inpainting"
```

Comment:
Let's name it "image-inpainting" to keep things aligned. 
```suggestion
    INPAINTING = "image-inpainting"
```